.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "dart.sdkPath": "/usr/local/Cellar/dart/1.24.3/libexec"
```

Comment:
Do we want to have these settings in the to repo?

Reply:
nope